src/components/Waveform/Waveform.tsx

```diff
@@ -197,6 +202,10 @@ export const Waveform = forwardRef<IWaveformRef, IWaveform>((props, ref) => {
   const startPlayerAction = async (args?: IStartPlayerRef) => {
     if (mode === 'static') {
       try {
+        if (playerState === PlayerState.stopped) {
+          await preparePlayerForPath();
```

Comment:
@nilesh-simform  Why did we need to add this here? Once the audio component mounts, I think we are preparing the player.

Reply:
I think we are doing this for `audioPlayers[key] = null // Release the player after stopping it` we have added when player stops, right?

Reply:
Yes, we are releasing the player when we stop it, due to limitation of 32 audio instance in android, so that's why we are again calculating it when we play audio again